data/sea/52-psv.h

```diff
@@ -71,7 +71,21 @@ static ifleet_t * INLINE psv_alloc_fleet (iint_t max_chord_count);
 
 static void INLINE psv_collect_fleet (ifleet_t *fleet);
 
-static ierror_loc_t INLINE psv_configure_fleet (const char *entity, size_t entity_size, const ichord_t **chord, ifleet_t *fleet);
+#if ICICLE_ZEBRA
+static ierror_loc_t INLINE psv_configure_fleet
+  ( const char *entity
+  , size_t entity_size
+  , piano_t *piano
+  , ifleet_t *fleet
+  );
+#else
+static ierror_loc_t INLINE psv_configure_fleet
+  ( const char *entity
+  , size_t entity_size
+  , const ichord_t **chord
+  , ifleet_t *fleet
+  );
+#endif
```

Comment:
Unless it's easier to do it this way for now, we should be able to just delete all the `ichord_t` stuff, piano is designed to handle the ice chord use case as well